src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/ConditionLabel.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <label
+    class="condition px-1"
+    :class="{
+      editable: editable,
+    }"
+    :contenteditable="editable"
+    @dblclick="handleDblClick"
+    @blur="handleBlur"
+    @keyup="handleKeyup"
+    @keydown="handleKeydown"
+  >
+    {{ editable ? conditionText : `[${conditionText}]` }}
+  </label>
+</template>
+<script>
+import { computed, nextTick, ref } from "vue";
+import { useStore } from "vuex";
+
+export default {
+  name: "ConditionLabel",
+  props: ["block", "getConditionFromBlock"],
+  setup(props) {
+    const editable = ref(false);
+    const store = useStore();
+    const code = computed(() => store.getters.code);
+    const onContentChange = computed(
+      () => store.getters.onContentChange || (() => {}),
+    );
+    const condition = computed(() => props.getConditionFromBlock(props.block));
+    const conditionText = computed(
+      () => condition?.value?.getFormattedText() ?? "",
+    );
+
+    function toggleEditable(_editable) {
```

Comment:
The name 'toggleEditable' suggests an automatic switch of states. It would be clearer to either revise the function to do that, or rename it to 'setEditable' to reflect the current behavior of setting a value.
```
function toggleEditable() {
  editable.value = !editable.value
}
```